server/protocols/webadmin-cli/README.md

```diff
@@ -148,7 +148,7 @@ If you want to update a user's password, please have a look at [Update a user pa
 ### Update a user password
 
 ```
-{cli} user update <username> --password
+{cli} user create --force <username> --password
```

Comment:
Do we need code changes?

Reply:
yes, I will open a ticket JAMES 3400.... later

Reply:
or maybe i just should do it right in this ticket

Reply:
Your change impacts the CLI, you should handle thiat in this pull request.